### PR TITLE
Update cameras.xml

### PR DESCRIPTION
**Pull request by PeterWem**
Added white points/ISO for 1Ds and renewed its color matrix. Adobe DNG Converter 14.2 was used.
This PR solves https://github.com/darktable-org/rawspeed/issues/375

### CODE REVIEW

File: data/cameras.xml [hunk 1]
```diff
@@ -1784,12 +1784,21 @@
 			<Color x="1" y="1">GREEN</Color>
 		</CFA>
 		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="0" white="3500"/>
-		<ColorMatrices>
-			<ColorMatrix planes="3">
-				<ColorMatrixRow plane="0">4374 3631 -1743</ColorMatrixRow>
-				<ColorMatrixRow plane="1">-7520 15212 2472</ColorMatrixRow>
-				<ColorMatrixRow plane="2">-2892 3632 8161</ColorMatrixRow>
+		<Sensor black="0" white="3617"/>
```
Comment (kmilos) [inline data/cameras.xml]: I think this should go last (fallback value if ISO not matched): https://github.com/darktable-org/rawspeed/tree/develop/data#sensor-info

**Comment by LebedevRI** [comment]
If somebody really cares about this, just implement reading of those values from the metadata.

**Comment by PeterWem** [comment]
Hm, I will close this PR, redo it and make it less messy. 

Reading from Exif would be the best solution. It seems Canon has the black/white point in many different tags depending on camera model?

**Comment by kmilos** [comment]
Wrt Canon, black and white levels are indeed all over the place depending on MakerNote version: https://exiftool.org/TagNames/Canon.html

**Pull request closed without merging**